src/components/LoginForm.js

```diff
@@ -0,0 +1,110 @@
+import React, {Component} from 'react';
+import {
+  View,
+  Text,
+  KeyboardAvoidingView,
+  StyleSheet,
+  TextInput,
+  TouchableOpacity,
+} from 'react-native';
+
+export default class LoginForm extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {email: '', password: ''};
+  }
+
+  render() {
+    return (
```

Comment:
Use destructuring instead of accessing the state directly every time.

```suggestion
    const { password, email } = this.state;
    return (
```